mongo/filterstore.go

```diff
@@ -1,10 +1,21 @@
 package mongo
 
 import (
+	"errors"
 	"github.com/ONSdigital/dp-filter-api/models"
+	"github.com/satori/go.uuid"
 	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
 )
 
+const Database = "filters"
+const Filters_Collection = "filters"
```

Comment:
Variable names should be camelCase and if you want these exported use PascalCase (a.k.a. upper camel case).

Definitely no underscores.